src/System.ComponentModel.TypeConverter/src/System/ComponentModel/BaseNumberConverter.cs

```diff
@@ -45,16 +42,20 @@ public abstract class BaseNumberConverter : TypeConverter
         /// </summary>
         public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
         {
-            return sourceType == typeof(string) || base.CanConvertFrom(context, sourceType);
+            if (sourceType == typeof(string))
+            {
+                return true;
+            }
+
+            return base.CanConvertFrom(context, sourceType);
```

Comment:
Why did you change this code?  The previous style seems perfectly fine.

Reply:
Well the line of code didnt fit on my screen so i wanted to format it. Reverted